pkg/virt-controller/services/template.go

```diff
@@ -282,23 +282,6 @@ func generateQemuTimeoutWithJitter(qemuTimeoutBaseSeconds int) string {
 	return fmt.Sprintf("%ds", timeout)
 }
 
-func computePodSecurityContext(vmi *v1.VirtualMachineInstance, seccomp *k8sv1.SeccompProfile) *k8sv1.PodSecurityContext {
-	psc := &k8sv1.PodSecurityContext{}
-
-	if util.IsNonRootVMI(vmi) {
-		nonRootUser := int64(util.NonRootUID)
-		psc.RunAsUser = &nonRootUser
-		psc.RunAsGroup = &nonRootUser
```

Comment:
Currently, the security context for container-disks does not specify `RunAsGroup`:

https://github.com/kubevirt/kubevirt/blob/358086ceb2bdd4d404c402e56c8a2cad3f9b9cb0/pkg/container-disk/container-disk.go#L340-L347

If the pod security context is dropped, then `RunAsGroup` will not be inherited by the containers and I guess the group ID will be set to `0`. Probably this needs to be set explicitly and also worth checking other use cases.

Reply:
I added the group id and checked all other containers they use the `rendercontainer.go` `ContainerSpecRenderer`
https://github.com/kubevirt/kubevirt/blob/358086ceb2/pkg/virt-controller/services/rendercontainer.go#L184-L199
 
or `virtiofs`
https://github.com/kubevirt/kubevirt/blob/358086ceb2/pkg/virt-controller/services/virtiofs.go#L77-L87

or container-disk which should now also set both user id and group id.

so I think now all of them set user id and group id.